configs/mender_convert_config

```diff
@@ -230,6 +230,10 @@ IMAGE_ROOTFS_EXTRA_SPACE="0"
 # why it is missing a "MENDER_" prefix.
 IMAGE_OVERHEAD_FACTOR="1.5"
 
+# User specified mount point for the boot partition. If empty (default), is mounted
+# at '/boot/efi' for UEFI integration, or at '/uboot' for other cases (U-boot)
```

Comment:
Question:
Is U-boot the only other case? It sounds like it from the description, but `cases` is plural

Reply:
Yeah, I didn't know how to phrase it well. Is technically speaking "all other cases", as the logic is if EFI this, else /uboot. So in a way we assume that if it is not EFI is uboot, but I guess it could be other case being whatever boot-loader and we still mount it in /uboot dir